PHP-Push-2: Updated to Z-Push 2.1.1 and modified Caldav Backend

tkoester opened the pull request:
Hi dupondje,

first of all - this is my first pull request ever. So please excuse me if something's not quite as it should be. 
I stumbled upon PHP-Push-2 when I tried to get a Baikal server to work with Z-Push. It turned out that almost everything went really smooth (thanks!), apart from the CalDav backend. For my server setup, I couldn't get authentication to work. After some research I decided to re-write the caldav-client-v2.php using ideas from carddav.php.
I hope that my modifications are useful for other people, too. Since your repository is as "upstream" as I could find, I trust that you are the maintainer of this package. If this is so, I would be glad if you could consider my pull request for inclusion. If you're not the person responsible, please point me in the right direction. Many thanks in advance!

Best,
tkoester

tkoester committed:
Rebase Z-Push 2.1.0-1750

tkoester committed:
Rebase Z-Push 2.1.1-1788

tkoester committed:
updated overriding functions from BackendDiff

tkoester committed:
added include/caldav.php based on include/caldav-client-v2.php, but with cURL mechanics borrowed from include/carddav.php

dupondje commented:
Hi.

Thanks alot for the pull request.
Since yesterday my code is also part of the 'official' z-push repository. So thats the new "upstream" now in fact.
http://svn.z-push.org/listing.php?repname=Z-Push

My code was already modified a bit by @fmbiete. So maby he can comment/include your changes.

This as I don't use the PHP-Push myself anymore, I don't really test/debug it any longer.

tkoester commented:
Hi,

thanks for the quick reply, and congratulations on getting your code included in the official z-push repository. Acting upon your answer, I already forwarded my changes to @fmbiete, so I think I'll better close this pull request now.

tkoester commented:
Hi,

is it OK with you if I license my derived caldav.php under LGPL v3? I re-used the cURL part of your carddav.php (apparently under AGPL v3) for modifying AWL’s caldav-client-v2.php, which is licensed under LGPL v3. I’m a bit unsure as to what to do about such a combination of licenses, so I thought I’d better ask…

Best,

tkoester

Von: dupondje [mailto:notifications@github.com] 
Gesendet: Montag, 31. März 2014 20:51
An: dupondje/PHP-Push-2
Cc: Thorsten Köster
Betreff: Re: [PHP-Push-2] PHP-Push-2: Updated to Z-Push 2.1.1 and modified Caldav Backend (#94)

Hi.

Thanks alot for the pull request.
Since yesterday my code is also part of the 'official' z-push repository. So thats the new "upstream" now in fact.
http://svn.z-push.org/listing.php?repname=Z-Push

My code was already modified a bit by @fmbiete https://github.com/fmbiete . So maby he can comment/include your changes.

This as I don't use the PHP-Push myself anymore, I don't really test/debug it any longer.

—
Reply to this email directly or view it on GitHub https://github.com/dupondje/PHP-Push-2/pull/94#issuecomment-39126344 .  https://github.com/notifications/beacon/7046561__eyJzY29wZSI6Ik5ld3NpZXM6QmVhY29uIiwiZXhwaXJlcyI6MTcxMTkxMTA2NSwiZGF0YSI6eyJpZCI6Mjg5Mjk3OTJ9fQ==--468298f55c603234db344299d572bea544355a0c.gif
